### PR TITLE
chore: replace sh with bash for syntax highlighting

### PR DESCRIPTION
**Pull request by craicoverflow**
Replace `sh` with `bash` everywhere for better syntax highlighting in Docusaurus.

### CODE REVIEW

File: docs/graphql-migrations/intro.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ You can install `graphql-migrations` on your existing project using the followin
...
 
-```shell
+```bashell
 npm i graphql-serve
 ```
 
 or with yarn:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphql-migrations/intro.md:27]:
```suggestion
```bash
```

File: docs/graphql-migrations/intro.md [hunk 2]
```diff
@@ -18,13 +18,13 @@ You can install `graphql-migrations` on your existing project using the followin
 
 With npm: 
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphql-migrations/intro.md:21]:
```suggestion
```bash
```

File: docs/graphqlserve/graphqlserve.md [hunk 3]
```diff
@@ -12,19 +12,19 @@ GraphQL Serve is a CLI tool that leverages the power of Graphback to generate a
...
 
-```shell
+```bashell
 npm i -g graphql-serve
 ```
 
 or with yarn:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphqlserve/graphqlserve.md:21]:
```suggestion
```bash
```

File: docs/graphqlserve/graphqlserve.md [hunk 4]
```diff
@@ -12,19 +12,19 @@ GraphQL Serve is a CLI tool that leverages the power of Graphback to generate a
...
 yarn global add graphql-serve
 ```
 
 or run it with [npx](https://www.npmjs.com/package/npx): 
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphqlserve/graphqlserve.md:27]:
```suggestion
```bash
```

File: docs/graphqlserve/graphqlserve.md [hunk 5]
```diff
@@ -48,27 +48,27 @@ The `@model` annotation indicates that `Note` is a data model and Graphback will
 
 To start your server, run the following command from the same directory as `Note.graphql`:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphqlserve/graphqlserve.md:51]:
```suggestion
```bash
```

File: docs/graphqlserve/graphqlserve.md [hunk 6]
```diff
@@ -48,27 +48,27 @@ The `@model` annotation indicates that `Note` is a data model and Graphback will
...
 gqlserve serve Note.graphql
 ```
 
 This will start a GraphQL server on a random port using the `Note.graphql` data models we just added.
 
 You can customise the directory of the data models:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphqlserve/graphqlserve.md:59]:
```suggestion
```bash
```

File: docs/graphqlserve/graphqlserve.md [hunk 7]
```diff
@@ -48,27 +48,27 @@ The `@model` annotation indicates that `Note` is a data model and Graphback will
...
 gqlserve serve ./path/to/models
 ```
 
 You can also specify where to load the data models from with a Glob pattern:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphqlserve/graphqlserve.md:65]:
```suggestion
```bash
```

File: docs/graphqlserve/graphqlserve.md [hunk 8]
```diff
@@ -48,27 +48,27 @@ The `@model` annotation indicates that `Note` is a data model and Graphback will
...
 gqlserve serve ./schema/**/*.graphql
 ```
 
 You can specify which port to start the server on:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphqlserve/graphqlserve.md:71]:
```suggestion
```bash
```

File: docs/graphqlserve/graphqlserve.md [hunk 9]
```diff
@@ -104,7 +104,7 @@ type Note {
 
 Once we have a model with datasync capabilities, we can run our GraphQL server by enabling data synchronization as shown below:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphqlserve/graphqlserve.md:107]:
```suggestion
```bash
```

File: docs/graphqlserve/graphqlserve.md [hunk 10]
```diff
@@ -114,7 +114,7 @@ Graphback receives your data models as an input and processes them to generate a
 
 GraphQL Serve allows you to print the resulting schema in your terminal with the `print-schema` subcommand:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphqlserve/graphqlserve.md:117]:
```suggestion
```bash
```

File: docs/graphqlserve/graphqlserve.md [hunk 11]
```diff
@@ -125,7 +125,7 @@ Generated schema:
 
 This information is also provided with the command itself:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphqlserve/graphqlserve.md:128]:
```suggestion
```bash
```

File: docs/graphqlserve/graphqlserve.md [hunk 12]
```diff
@@ -142,7 +142,7 @@ Options:
 
 For the `serve` command:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphqlserve/graphqlserve.md:145]:
```suggestion
```bash
```

File: docs/graphqlserve/graphqlserve.md [hunk 13]
```diff
@@ -164,7 +164,7 @@ Examples:
 
 Also for `print-schema` command:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/graphqlserve/graphqlserve.md:167]:
```suggestion
```bash
```

File: packages/graphql-serve/README.md [hunk 14]
```diff
@@ -30,13 +30,13 @@ database.
 
 Using npm:
 
-```shell
+```bashell
```
Comment (machi1990) [inline packages/graphql-serve/README.md:33]:
```suggestion
```bash
```

File: packages/graphql-serve/README.md [hunk 15]
```diff
@@ -30,13 +30,13 @@ database.
...
 npm install -g graphql-serve
 ```
 
 or yarn:
 
-```shell
+```bashell
```
Comment (machi1990) [inline packages/graphql-serve/README.md:39]:
```suggestion
```bash
```

File: packages/graphql-serve/README.md [hunk 16]
```diff
@@ -60,20 +60,20 @@ type Note {
 
 Assuming you have created your various `*.graphql` data model files in the `models` directory, to automatically generate resolvers and start a GraphQL server listening on port 8080 do the following:
 
-```shell
+```bashell
```
Comment (machi1990) [inline packages/graphql-serve/README.md:63]:
```suggestion
```bash
```

File: packages/graphql-serve/README.md [hunk 17]
```diff
@@ -60,20 +60,20 @@ type Note {
...
 $ gqlserve serve models --port=8080
 ```
 
 
 If you only need to see the generated GraphQL Schema, use the `print-schema` command:
 
-```shell
+```bashell
```
Comment (machi1990) [inline packages/graphql-serve/README.md:70]:
```suggestion
```bash
```

File: packages/graphql-serve/README.md [hunk 18]
```diff
@@ -60,20 +60,20 @@ type Note {
...
 $ gqlserve print-schema .
 ```
 The above command prints schema generated from data model files in the current directory.
 
 This information is also provided with the command itself:
-```shell
+```bashell
```
Comment (machi1990) [inline packages/graphql-serve/README.md:76]:
```suggestion
```bash
```

File: packages/graphql-serve/README.md [hunk 19]
```diff
@@ -88,7 +88,7 @@ Options:
   -v, --version  Show version number                                   [boolean]
 ```
 For the `serve` command:
-```shell
+```bashell
```
Comment (machi1990) [inline packages/graphql-serve/README.md:91]:
```suggestion
```bash
```

File: packages/graphql-serve/README.md [hunk 20]
```diff
@@ -109,7 +109,7 @@ Examples:
 ```
 
 Also for `print-schema` command:
-```shell
+```bashell
```
Comment (machi1990) [inline packages/graphql-serve/README.md:112]:
```suggestion
```bash
```

File: templates/ts-apollo-postgres-backend/.graphqlrc.yaml [hunk 21]
```diff
@@ -0,0 +1,9 @@
+schema: './src/schema.graphql'
+documents: ./client/src/graphql/**/*.graphql
+extensions:
+  graphback:
+    model: ./model/datamodel.graphql
+    plugins:
+      graphback-client:
+        format: 'ts'
+        outputFile: ./client/src/graphql/graphback.ts
```
Comment (machi1990) [inline templates/ts-apollo-postgres-backend/.graphqlrc.yaml:9]: Unrelated change

File: templates/ts-react-apollo-client/src/graphql/graphback.ts [hunk 22]
```diff
@@ -0,0 +1,238 @@
...
+      text
+      description
+   }
+} 
+`
+
+
+
+
+export const CommentFragment = gql`
```
Comment (machi1990) [inline templates/ts-react-apollo-client/src/graphql/graphback.ts:29]: Unrelated change

File: docs/databases/mongodb.md [hunk 23]
```diff
@@ -13,13 +13,13 @@ The package is built on top of the official [MongoDB Node.js Driver](https://mon
...
 
-```shell
+```bashell
 npm install @graphback/runtime-mongo mongodb
 ```
 
 Install with yarn:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/databases/mongodb.md:22]:
```suggestion
```bash
```

File: docs/databases/mongodb.md [hunk 24]
```diff
@@ -13,13 +13,13 @@ The package is built on top of the official [MongoDB Node.js Driver](https://mon
 
 Install with npm:
 
-```shell
+```bashell
```
Comment (machi1990) [inline docs/databases/mongodb.md:16]:
```suggestion
```bash
```

**Comment by craicoverflow** [comment]
Thanks for noticing those changes @machi1990 !

**Comment by machi1990** [comment]
> Thanks for noticing those changes @machi1990 !

All good, except for this comment https://github.com/aerogear/graphback/pull/1694#discussion_r452382817 not sure if you seen it?

**Comment by craicoverflow** [comment]
I had seen it, and removed the file - but then did not commit that removal! Gone now.

**Comment by machi1990** [comment]
Nice work. It will make the docs better! Thanks @craicoverflow
